icu4c/source/samples/coll/coll.cpp

```diff
@@ -244,12 +244,12 @@ UBool processCollator()
  */
 int main(int argc, const char** argv) 
 {
-    if (processOptions(argc, argv, opts) != TRUE || opt_help) {
+    if (static_cast<bool>(processOptions(argc, argv, opts)) != true || static_cast<bool>(opt_help)) {
```

Comment:
Kind of a nit, but do we still need to compare equal/not equal to true here, versus using/not using `!` (same in other places/files)?

Reply:
Oh good point, since these are `bool`s now, we should be able to just use an if statement. :) 
I'll force-push an update later.